popde/density_estimate.py

```diff
@@ -1,3 +1,92 @@
+import numpy as np
+
+class SimpleKernelDensityEstimation:
+    """
+    Fit and evaluate general Kernel Density Estimation (KDE) 
+
+    Methods:
+    --------
+    fit(data, bandwidth=None, kernel=None):
+        Fit the KDE to new data.
+
+    evaluate(points):
+        Evaluate the KDE at given data points.
+
+    Examples:
+    ---------
+    # Create a KDE instance and fit it to data
+    data = np.random.randn((100, 100))
+    kde = SimpleGaussianKernelDensityEstimation(data)
+
+    # Evaluate the KDE at new data points
+    new_data = np.array(np.linspace(-3, 3, 100).tolist(), np.linspace(-3, 3, 100).tolist())
+    density_values = kde.evaluate(new_data)
+    """
+
+    def __init__(self, data, bandwidth=1.0, kernel='gaussian', dim_names=None):
+        """
+        Initialize the KernelDensityEstimation object.
+
+        Parameters:
+        -----------
+        data : array-like
```

Comment:
We should make a decision and document what the shape should be, either (nevents, ndim) or (ndim, nevents)